arch/arm/soc/st_stm32/stm32f4/Kconfig.defconfig.stm32f413xx

```diff
@@ -0,0 +1,18 @@
+# Kconfig - ST STM32F413XH MCU configuration options
+#
+# Copyright (c) 2017 Florian Vaussard, HEIG-VD
```

Comment:
We're not using names in copyright, but I don't know if this is a real rule.
I know some do, but some other have been rejected.
@galak, you might know if there is a strict rule?